pkg/config/template.go

```diff
@@ -1103,6 +1106,12 @@ const templateStringCrioRuntimeRuntimesRuntimeHandler = `# The "crio.runtime.run
 {{ $.Comment }}allowed_annotations = [
 {{ range $opt := $runtime_handler.AllowedAnnotations }}{{ $.Comment }}{{ printf "\t%q,\n" $opt }}{{ end }}{{ $.Comment }}]
 {{ $.Comment }}{{ end }}
+{{ $.Comment }}monitor_path = "{{ $runtime_handler.MonitorPath }}"
+{{ $.Comment }}{{ if $runtime_handler.MonitorEnv }}
+{{ $.Comment }}monitor_env = [
+{{ range $opt := $runtime_handler.MonitorEnv }}{{ $.Comment }}{{ printf "\t%q,\n" $opt }}{{ end }}{{ $.Comment }}]
+{{ $.Comment }}{{ end }}
+{{ $.Comment }}monitor_cgroup = "{{ $runtime_handler.MonitorCgroup }}"
```

Comment:
The new option names are missing the runtime_ prefix to align with the declarations in config.go

Reply:
we already have privileged_without_host_devices not prefixed. I personally think it's alright, as these fields are configuring the monitor, not the runtime

Reply:
Aren't the new options defined with a runtime_ prefix in config.go, though? ie. runtime_monitor_path? So the example config this template generates would be missing that prefix and not be parsed?

Reply:
oh jeez, you're right. I'll fix it there

Reply:
fixed